### ISSUE TITLE
Implement functionality for shorthand HEX format

### ISSUE THREAD

**Issue by AndreyDmitriev39r** [DBenMoshe/contrast-ratio-repo#77]
User should be able to get contrast ratio for two colors in format like below:
```
#3FA
#FAC
```
Example of behaviour:
![17 08 2022_00 01 29_REC](https://user-images.githubusercontent.com/108119145/184993324-cef4218e-d673-43af-8c41-51814b495ddc.png)

To get some ideas about approach, look at existing functionality for two HEX colors, and also feel free to ask questions.

**Comment by Okibe00** [comment]
Looks like we can just convert it into a six digit hex and the functions you created already takes care of the rest, at least that's what I'm thinking.

**Comment by AndreyDmitriev39r** [comment]
Sounds good, good luck with this one!
